apps/locale/es_AR/book-2.app

```diff
@@ -2,14 +2,4 @@
     "title": "Libro Kano 2",
     "tagline": "Poderes de programación",
     "description": "Este es un increíble libro acerca del sistema operativo de Kano. Ábrelo para conocer la historia de Ping Pong, Minecraft y más, también encontrarás un montón de información acerca de programación y juegos con Kano.",
-    "slug": "kano-book-2",
-
-    "icon": "book2",
-    "colour": "#6B7678",
-
-    "categories": ["media"],
-
-    "packages": [],
-    "dependencies": ["epdfview"],
-    "launch_command": "epdfview '/usr/share/kano-apps/books/Book 2 - Levels.pdf'"
```

Comment:
Don't we need this here similar to Book 1? It's because of the install path I guess